src/components/views/project/ProjectDonateCard.tsx

```diff
@@ -255,11 +256,20 @@ const ProjectDonateCard = ({
 				</BadgeWrapper>
 				{!isDraft && (
 					<GivBackNotif>
-						<GLink size='Medium' color={brandColors.giv[300]}>
+						<Caption color={brandColors.giv[300]}>
 							When you donate to verified projects, you get GIV
 							back.
-						</GLink>
-						<QuestionBadge />
+						</Caption>
+						<a
+							href='https://docs.giveth.io/giveconomy/givbacks'
```

Comment:
We may change the links in the future, or we may wanna use them in other pages. So I suggest add this url as "GIVBACK_DOCS" to the links file (lib/constants/links.ts).

Reply:
great idea,
Thank you @RamRamez .